corehq/apps/es/client.py

```diff
@@ -981,10 +984,8 @@ def from_python(self, doc):
     # meta methods and Elastic index read methods (pass-through on the primary
     # adapter)
     @property
-    def settings(self):
```

Comment:
Don't we need this property for Multiplexed Adapter?

Reply:
No, the settings no longer live as a "static" property on the adapter. Instead, the adapter keeps a "settings_key", which is used to render the settings data when it is needed.